README.md

```diff
@@ -502,6 +528,44 @@ The funds transfers feature enables you send money directly from your paystack b
 
 ```
 
+## Refunds
+
+### Create a refund
+
+```ruby
+
+ refund = PaystackRefunds.new(paystackObj)
+
+ #full refund
+ transaction = {transaction: transaction_reference}
+ refund.createRefund(transaction) #must be a valid transaction reference or transaction id
+
+ #partial refund
+ transaction = {transaction: transaction_reference, amount: 5000} # minimum amount 50 NGN or 5000 kobos
+ refund.createRefund(transaction)
+
+```
+
+### List refunds
+
+```ruby
+
+	page_number = 1
+ 	refund = PaystackRefunds.new(paystackObj)
+	refund.list(page_number) # page_number is optional
+
+```
+
+
+### Fetch refund
+
+```ruby
+
+ 	refund = PaystackRefunds.new(paystackObj)
+	refund.get(refund_id) # returned at the time of creating a refund
+
+```
+
 ## Static methods
 `PaystackTransactions`, `PaystackCustomers`, `PaystackPlans`, `PaystackSubaccounts`, `PaystackBanks` , `PaystackSubscriptions` , `PaystackSettlements`, `PaystackBalance`, and `PaystackTransfers` methods can be called statically, You just need to pass the paystack object as the first parameter  e.g. `verify` method in `PaystackTransactions` can be called like this
```

Comment:
you are missing the `PaystackRefunds` class in this list

Reply:
@arisah-ijeoma Can you please describe where is the `PaystackRefunds` class is not included in the list.

Reply:
@mutafaf you have updated the codebase, and the readme. Going by the work you did, it would seem that PaystackRefunds should be part of Static methods but it isn't